README.md

```diff
@@ -479,6 +479,48 @@ Contributions of additional uses cases on top of ECS are welcome.
 * *Use prefixes.* Fields must be prefixed except for the base fields. For example all `host` fields are prefixed with `host.`. See `dot` notation in FAQ for more details.
 * Do not use abbreviations. (A few exceptions like `ip` exist.)
 
+## Understanding ECS conventions
+
+### Multi-fields text indexing
+
+ElasticSearch can index text multiple ways:
+
+* `text` indexing allows for full text search, or searching arbitrary words that
+  are part of the field.
+* `keyword` indexing allows for exact match search (much faster) and allows for
+  aggregations (what Kibana visualizations are built on).
+
+In some cases only one type of indexing makes sense for a field. E.g. no need to
+do full text search on an id, and nobody needs to do an exact match search on
```

Comment:
`normally there is no need for an exact match search on 2 kb stack trace`. Nobody is a bit strong :-)